packages/react/src/vertex3.tsx

```diff
@@ -0,0 +1,198 @@
+import { Utility } from "@hpcc-js/common";
+import * as React from "@hpcc-js/preact-shim";
+import { Icon } from "./icon";
+import { TextBox } from "./text";
+import { Vertex } from "./vertex";
+
+export interface IVertex3 extends Vertex {
+    categoryID?: string;
+    text: string;
+    textHeight?: number;
+    textPadding?: number;
+    textboxStrokeWidth?: number;
+    icon?: Icon;
+    annotationsHeight?: number;
+    annotations?: string[];
+    annotationMeta?: TextBox[];
+    annotationGutter?: number;
+    textFill?: string;
+    textboxFill?: string;
+    textboxStroke?: string;
+    textFontFamily?: string;
+    cornerRadius?: number;
+    subText?: TextBox;
+    onSizeUpdate?: (size: { width: number, height: number }) => void;
+}
+
+export const Vertex3: React.FunctionComponent<IVertex3> = ({
+    categoryID = "",
+    text = "",
+    textHeight = 10,
+    textPadding = 4,
+    textFill = "#287EC4",
+    textboxFill = "white",
+    textboxStroke = "#CCCCCC",
+    textboxStrokeWidth = 1,
+    textFontFamily = "Verdana",
+    annotationGutter = 2,
+    annotationMeta = [],
+    cornerRadius = 3,
+    icon = {},
+    subText = {}
+}) => {
+    icon = {
+        height: 50,
+        imageChar: "?",
+        imageFontFamily: "'Font Awesome 5 Free'",
```

Comment:
Should default to "FA 4.7"

Reply:
Why not use the latest? We're leaving so many icons behind with 4.7.

Reply:
I am good with that - but there is more involved than referencing it here!!!